[2024-06-30 18:35:03,201] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.insert_into_datalake manual__2024-06-30T18:34:59.375493+00:00 [queued]>
[2024-06-30 18:35:03,206] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.insert_into_datalake manual__2024-06-30T18:34:59.375493+00:00 [queued]>
[2024-06-30 18:35:03,206] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 18:35:03,206] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-06-30 18:35:03,206] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 18:35:03,213] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): insert_into_datalake> on 2024-06-30 18:34:59.375493+00:00
[2024-06-30 18:35:03,217] {standard_task_runner.py:52} INFO - Started process 508 to run task
[2024-06-30 18:35:03,219] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_datalake_dag', 'insert_into_datalake', 'manual__2024-06-30T18:34:59.375493+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpq41kqszl', '--error-file', '/tmp/tmp0ent1qp3']
[2024-06-30 18:35:03,219] {standard_task_runner.py:80} INFO - Job 4: Subtask insert_into_datalake
[2024-06-30 18:35:03,251] {task_command.py:371} INFO - Running <TaskInstance: example_datalake_dag.insert_into_datalake manual__2024-06-30T18:34:59.375493+00:00 [running]> on host 12d6cc628018
[2024-06-30 18:35:03,326] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_datalake_dag
AIRFLOW_CTX_TASK_ID=insert_into_datalake
AIRFLOW_CTX_EXECUTION_DATE=2024-06-30T18:34:59.375493+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-30T18:34:59.375493+00:00
[2024-06-30 18:35:03,348] {base.py:68} INFO - Using connection ID 'datalake_postgres' for task execution.
[2024-06-30 18:35:03,356] {dbapi.py:231} INFO - Running statement: 
        INSERT INTO datalake_table (data) VALUES
        ('example_data');
        , parameters: None
[2024-06-30 18:35:03,359] {dbapi.py:239} INFO - Rows affected: 1
[2024-06-30 18:35:03,366] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=example_datalake_dag, task_id=insert_into_datalake, execution_date=20240630T183459, start_date=20240630T183503, end_date=20240630T183503
[2024-06-30 18:35:03,409] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-06-30 18:35:03,440] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
